doc/src/stdlib/arrays.md

```diff
@@ -7,15 +7,19 @@ Core.AbstractArray
 Base.AbstractVector
 Base.AbstractMatrix
 Core.Array
-Core.Array(::Any)
-Core.Array(::Any, ::Any)
+Core.Array(::Uninitialized, ::Any)
```

Comment:
Where these not included in the `Array` entry on the line above?

Reply:
No, I got an error:
```
 !! No docs found for 'Core.Array(::Any, ::Any)'. [src/stdlib/arrays.md]
```
I thought there was a way to include all methods, but this one doesn't work at least.

Reply:
Right, I meant that you should remove `Core.Array(::Any)` and `Core.Array(::Any, ::Any)` and keep `Core.Array`. I *think* that `Core.Array` should catch everything associated with `Array`, but perhaps there was some reason for including them all earlier, perhaps to place the type docstring for `Array` at the top?

Reply:
No, that only includes the type, not methods.

Reply:
Ok, good to know!